Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift

```diff
@@ -2,16 +2,69 @@
 // Licensed under the MIT License.
 
 import UIKit
+import AppCenterDistribute
 
 class MSDistributeViewController: UITableViewController, AppCenterProtocol {
 
   @IBOutlet weak var enabled: UISwitch!
   @IBOutlet weak var customized: UISwitch!
+  @IBOutlet weak var preStartSwitch: UISwitch!
+  @IBOutlet weak var updateTrackField: UITextField!
   var appCenter: AppCenterDelegate!
-  
+
+  enum UpdateTrack: String {
```

Comment:
Can't we reuse SDK enum here?

Reply:
This enum is only used for converting UpdateTrack to a string and displaying it in MSEnumPicker. It is unlikely to have any use outside of the current ViewController.

Reply:
![Simulator Screen Shot - iphone - 2020-01-27 at 14 13 57](https://user-images.githubusercontent.com/53528961/73170427-656a4680-410f-11ea-88a1-588ca683116d.png)